### DISCUSSION TITLE
vscode termiinal with zsh and starfish cant connect to codewhisperer socket after closing macbook lid and opening again

### DISCUSSION THREAD

**Discussion by igneous8k** [aws/codewhisperer-command-line-discussions#83] [Known bugs]
vscode termiinal with zsh and starfish cant connect to codewhisperer socket after closing macbook lid and opening again

**Comment by grant0417** [comment]
Hey @igneous8k can you share the error you are getting?

**Comment by igneous8k** [comment]
<img width="686" alt="image" src="https://github.com/aws/codewhisperer-command-line-discussions/assets/48514025/6251f059-e871-43db-91b3-d290943ba754">


✘ Cwterm: Socket exists but could not connect: Connection refused (os error 61)


✘ Doctor found errors. Please fix them and try again.

If you are not sure how to fix it, please open an issue with cw issue to let us know!

----------------------------------------------------------------------------------------
this only happens in vscode integrated terminal, outside iterm and default terminal works fine.

**Reply by brendanfalk** [reply]
Hey! This is super weird... Could you please email grangurv@amazon.com, mattxwhi@amazon.com, and bjf@amazon.com. We would love to get on a call to resolve this with you
